tests/std/tests/P0645R10_text_formatting_formatting/test.cpp

```diff
@@ -60,6 +60,45 @@ void throw_helper(const charT* fmt, const Args&... vals) {
     }
 }
 
+template <class back_inserter>
+struct move_only_back_inserter {
+    back_inserter it;

```

Comment:
let's use some other name for the template argument, or use capitals. Shadows a ::std:: function (thanks stl)